tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1569,6 +1569,41 @@ def _create_dummy_array_for_dot(xp, shape, dtype):
 ]
 
 
+_in_out_dtypes_math_binary_functions = dtype_utils._permutate_dtype_mapping([
```

Comment:
This line exceeds 70 characters.

Reply:
If I remember correctly, 82 is the limit in pep8 and flake8. Also the python style tests have passed. Are you sure it should be less than 70?

Reply:
Sorry, I was wrong about the line length. The limit is 79, in fact.